routing/localchans/manager.go

```diff
@@ -185,7 +185,7 @@ func (r *Manager) updateEdge(tx kvdb.RTx, chanPoint wire.OutPoint,
 	// Retrieve negotiated channel htlc amt limits.
 	amtMin, amtMax, err := r.getHtlcAmtLimits(tx, chanPoint)
 	if err != nil {
-		return nil
+		return err
```

Comment:
So this is wrong the whole time? We were not intentionally ignoring the error here right?